### PR TITLE
New profile for man,psi,smuxi; fix pidgin

### PR DESCRIPTION
**Pull request by kortewegdevries**
<s>Will add smuxi to this PR later...</s> Done I think...

### CODE REVIEW

File: etc/profile-m-z/psi.profile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+novideo
+nou2f
+protocol unix,inet,inet6,netlink
+seccomp !chroot
+shell none
+# breaks on Arch
+# tracelog
+
+disable-mnt
+# Add "gpg,gpg2,gpg-agent,pinentry,pinentry-gtk-2" for GPG
```
Comment (rusty-snake) [inline etc/profile-m-z/psi.profile:70]: Depending on DE other pinentry binaries are needed. Is pinentry for all distros a symlink? if so, the default binary is copied.

**Review by rusty-snake — COMMENTED** [review]
LGTM

I'm not sure whether we should add man to firecfg or not. Does it break man in root shells? I mean to remember such comments in other cli profiles.

File: etc/profile-m-z/psi.profile (hunk 1 above)
Comment (kortewegdevries) [reply etc/profile-m-z/psi.profile:70]: Not a symlink for me on Arch...

**Comment by kortewegdevries** [comment]
> Does it break man in root shells?

Working for me when I set a symlink using firecfg. `su`>`/usr/local/bin/man man`...

File: etc/profile-m-z/psi.profile (hunk 1 above)
Comment (rusty-snake) [reply etc/profile-m-z/psi.profile:70]: Can you add more pinentry binaries.

Comment (kortewegdevries) [reply etc/profile-m-z/psi.profile:70]:
```
pinentry-curses
pinentry-fltk
pinentry-gnome3
pinentry-gtk2 or pinentry-gtk-2
pinentry-qt
pinentry-qt4
pinentry-tty
pinentry-x2go
pinentry-kwallet 
```

Comment (rusty-snake) [reply etc/profile-m-z/psi.profile:70]: `pinentry-gtk` and `pinentry-emacs`(?)

Comment (kortewegdevries) [reply etc/profile-m-z/psi.profile:70]: I don't have a gtk; emacs one I forgot...

File: etc/profile-m-z/psi.profile [hunk 2]
```diff
@@ -67,7 +67,7 @@ shell none
 # tracelog
 
 disable-mnt
-# Add "gpg,gpg2,gpg-agent,pinentry,pinentry-gtk-2" for GPG
+# Add "gpg,gpg2,gpg-agent,pinentry-curses pinentry-emacs pinentry-fltk pinentry-gnome3 pinentry-gtk pinentry-gtk2 or pinentry-gtk-2 pinentry-qt pinentry-qt4 pinentry-tty pinentry-x2go pinentry-kwallet for GPG
```
Comment (rusty-snake) [inline etc/profile-m-z/psi.profile:70]:
```suggestion
# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for GPG
```

**Review by rusty-snake — APPROVED** [review]

**Comment by rusty-snake** [comment]
Thanks for this contribution!

**Pull request merged**